src/LiveComponent/assets/src/HookManager.ts

```diff
@@ -13,20 +9,15 @@ export default class {
 
     unregister(hookName: string, callback: () => void): void {
         const hooks = this.hooks.get(hookName) || [];
-
         const index = hooks.indexOf(callback);
-        if (index === -1) {
-            return;
+        if (index !== -1) {
+            hooks.splice(index, 1);
+            this.hooks.set(hookName, hooks);
         }
-
-        hooks.splice(index, 1);
-        this.hooks.set(hookName, hooks);
```

Comment:
Unless I’m missing something, it was better before: it was using an if to “exit earlier” -> less nested code